### DISCUSSION TITLE
识别结果输出的格式问题

### DISCUSSION THREAD

**Discussion by jweden33** [RapidAI/TableStructureRec#99] [Q&A]
目前识别结果的格式只有html格式，但是我想用json格式的结果，请问除了将html格式的结果转化为json格式外还有什么更好的方法吗

**Comment by Joker1212** [comment]
emmm,其实只是格式的转换的话， 都是规则的，不涉及到精度问题，因为json每个人使用场景不一样，所以这里没有统一给json的格式，建议是直接丢给大模型，让它帮你写转换的代码，你检查一下改改，也欢迎提pr来丰富输出哈

**Reply by jweden33** [reply]
好的明白了 谢谢大佬
